File: GoLang/8 kyu/Multiple of index/main.go
package main

import "fmt"

func main() {
	fmt.Println(multipleOfIndex([]int{22, -6, 32, 82, 9, 25}))
}

func multipleOfIndex (ints []int) (res []int) {
	for index := 1; index < len(ints); index++ {
	  
	  elem := ints[index]
	  
	  if elem%index == 0 {
		res = append(res, elem)
	  }
	  
	}
	return res
  }

File: Python/8 kyu/Multiple of index/main.py
from audioop import mul


def multiple_of_index(arr):
    ints = []
    for i in range(1, len(arr)):
        if arr[i]%i == 0:
            ints.append(arr[i])
    return ints

if __name__ == "__main__":
    print(multiple_of_index([22, -6, 32, 82, 9, 25]))